src/MOI_wrapper.jl

```diff
@@ -1850,7 +1850,7 @@ function MOI.get(
     column = _info(model, c).column
     x = get_dblattrelement(model.inner, "X", column)
     ub = get_dblattrelement(model.inner, "UB", column)
-    if x ≈ ub
+    if isapprox(x, ub, atol = get_dbl_param(model.inner, "FeasibilityTol"))
```

Comment:
What does `get_dblattrelement(model.inner, "RC", column)` return if `x` is different from `ub` ?

Reply:
See #250. It depends on the lower bound.